packages/babel-plugin-syntax-nullish-coalescing-operator/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "babel-plugin-syntax-nullish-coalescing-operator",
+  "version": "7.0.0-beta.3",
```

Comment:
any reason this can't just start at 1.0.0? I thought babel 7 was moving away from pegged version numbers.

Reply:
Just following precedent. @hzoo should this be changed?

Reply:
Yeah we need to move all proposals/stage-x presets into the new experimental folder and then version those separately